test/JDBC/expected/db_securityadmin-vu-verify.out

```diff
@@ -157,13 +159,49 @@ GO
 EXEC babel_5135_roleop_proc1;
 GO
 
--- CASE 3.2 - ADD/DROP the membership of user-defined database roles
+-- CASE 3.2.a - ADD/DROP the membership of user-defined database roles
 ALTER ROLE babel_5135_r1 ADD MEMBER babel_5135_u1;
 GO
 
 ALTER ROLE babel_5135_r1 DROP MEMBER babel_5135_u1;
 GO
 
+-- CASE 3.2.b - ADD/DROP the membership of user-defined database roles to/from current_user
+SELECT current_user;
+GO
+~~START~~
+varchar
+babel_5135_dbsecadmin_u1
+~~END~~
+
+
+ALTER ROLE babel_5135_r1 ADD MEMBER babel_5135_dbsecadmin_u1;
+GO
+
+ALTER ROLE babel_5135_r1 DROP MEMBER babel_5135_dbsecadmin_u1;
+GO
+
+-- CASE 3.2.c - It should be able to ALTER the membership of database role which is member of db_owner
+ALTER ROLE db_owner ADD MEMBER babel_5135_r1;
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Adding database roles to db_owner is not currently supported in Babelfish)~~
+
+
+ALTER ROLE babel_5135_r1 ADD MEMBER babel_5135_dbsecadmin_u1;
+GO
+
+ALTER ROLE babel_5135_r1 DROP MEMBER babel_5135_dbsecadmin_u1;
+GO
```

Comment:
Why is this test repeated? It is already there at line number 178 and 181.

Reply:
In this scenario, we wanted to test whether we can add current user as member of babel_5135_r1 (when it is member of db_owner).